src/app/job-routing/my-application/my-application.module.ts

```diff
@@ -0,0 +1,17 @@
+
+
+
+export class myApplication{
+
+        constructor( public companyLogo: string, public companyName: string, public role: string, public location: string, public salary: number, 
+                public tools: string[ ], public time: string){
+
+                    this.companyLogo = companyLogo;
+                    this.companyName = companyName
+                    this.role = role;
```

Comment:
If you are doing something like this, it is more advisable to implement an interface.. check. out Angular models in your tutorial and how to implement it in your code

Reply:
so, this interface should be created on the component(s) that need it?